partner_deduplicate_filter/tests/test_partner_deduplicate_filter.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2016 Tecnativa - Pedro M. Baeza
+# Copyright 2024 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import common
+from odoo.tools.safe_eval import safe_eval
+
+
+class TestDeduplicateFilter(common.TransactionCase):
```

Comment:
- Use BaseCommon as base test class to reduce overhead from tracking and boost the test suite.

```suggestion
from odoo.addons.base.tests.common import BaseCommon
from odoo.tools.safe_eval import safe_eval


class TestDeduplicateFilter(BaseCommon):
```